packages/postgres-database/src/simcore_postgres_database/models/functions_models_db.py

```diff
@@ -0,0 +1,181 @@
+"""Functions table
+
+- List of functions served by the simcore platform
+"""
+
+import uuid
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+from ._common import RefActions
+from .base import metadata
+
+functions = sa.Table(
+    "functions",
```

Comment:
maybe a good idea to prefix with `meta_modelling_` ?

Reply:
I can, although that i wouldn't consider functions as pure metamodelling

Reply:
well we are trying to have a bit of order in the database.. and have multiple services playing with these tables.

Reply:
ok, let me see how the other services differentiate their tables

Reply:
I will add funcapi_ to the table names